app/scripts/directives/resources.js

```diff
@@ -16,6 +16,53 @@ angular.module('openshiftConsole')
       }
     };
   })
+  .directive('podStatus', function($filter) {
+    return {
+      restrict: 'E',
+      scope: {
+        podStatus: '=',
+        initStatus: '=?',
+        terminal: '=?'
+      },
+      templateUrl: 'views/_pod-status.html',
+      link: function(scope) {
+        scope.isTerminal = angular.isFunction(scope.terminal);
```

Comment:
I feel like `isTerminal` is slightly confusing since the directive itself is not a terminal. Maybe `hasDebugTerminal` or `enableDebugTerminal`?